src/components/database/Queries.js

```diff
@@ -0,0 +1,345 @@
+// @flow
+import {database, fieldValue} from '../../firebaseFeatures.js';
+import type {UserType, PendingType, ActionType} from '../admin/FlowTypes.js';
+
+function sanitize(collection: string,
```

Comment:
Maybe this sanitize function would fit better in FlowTypes file?
From "any" type to "ActionType" conversion suits better (in my opinion) next to ActionType signature.